src/capi/types.h

```diff
@@ -139,7 +139,7 @@ class BoxedWrapperObject : public Box {
         if (flags == PyWrapperFlag_KEYWORDS) {
             wrapperfunc_kwds wk = (wrapperfunc_kwds)wrapper;
             rtn = (*wk)(self->obj, args, self->descr->wrapped, kwds);
-        } else if (flags == 0) {
+        } else if (flags == PyWrapperFlag_PYSTON || flags == 0) {
```

Comment:
is there a more general way we could handle this?  (ie what if we have `PyWrapperFlag_PYSTON | PyWrapperFlag_KEYWORDS`